testing/fuchsia/run_tests.sh

```diff
@@ -61,6 +64,7 @@ echo "$(date) START:WAIT_DEVICE_READY -------------------------------"
 for i in {1..10}; do
   ./fuchsia_ctl -d $device_name ssh \
       --identity-file $pkey \
+      --timeout-seconds $ssh_timeout_seconds \
```

Comment:
Same here the operation should complete in ms, with a faulty device this will block for 60mins